server/src/controller/trashController.js

```diff
@@ -0,0 +1,22 @@
+import validate from "../util/checkValidation.js";
+import prisma from "../../prisma/client.js";
+import { StatusCodes } from "http-status-codes";
+
+export const createTrash = async (req, res) => {
+    if (validate(req, res)) {
+      return res;
+    }
+    
+    const {weight, trashDate, categoryId, userId} = req.body;
```

Comment:
trashDate is not one of the body arguments listed in the trash.js routes file

Reply:
don't we not have to do the date because it is default to now?